datasets/doc/source/how-to.rst

```diff
@@ -0,0 +1,14 @@
+How-To Guides
+=============
+
+Flower Datasets library easily integrates with common frameworks like TensorFlow and PyTorch (among others) because it uses Hugging Face under the hood.
+Learn how to transform the HuggingFace dataset to the framework of your choice.
+
+.. toctree::
+   :maxdepth: 1
+
+   how-to-use-with-tf
+   how-to-use-with-torch
+   how-to-use-with-numpy
+
+Feel free to check the original HuggingFace `documentation <https://huggingface.co/docs/datasets/index>`_ if you didn't find the things you were looking for.
```

Comment:
This file can be removed. In framework/baselines, we link to individual how-to guides directly from `index.rst`

Reply:
Yes, but that's required to have a sidebar that can be clicked and toggled.

Reply:
The toggled sidebar hides the content by default, which is why we decided not to use it for framework and baselines docs. We'll use the same approach here.